csharp-basic-topics/LocalFunctionInCSharp/LocalFunctionInCSharp/Product.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace LocalFunctionInCSharp
+{
+    public class Product
+    {
+        public string Name { get; set; }
+        public int StockQuantity { get; set; }
+        public int SelectedQuantity { get; set; }
```

Comment:
All of these properties could be made init only, it is also a good candidate to be a record based on its usage.